package.json

```diff
@@ -6,7 +6,8 @@
   "type": "module",
   "scripts": {
     "test": "ava --verbose",
-    "lint": "standard"
+    "lint": "standard",
+    "deploy": "aws lambda update-function-code --cli-connect-timeout 15000 --region $AWS_REGION --function-name $LAMBDA --image $AWS_ACCOUNT_ID.dkr.ecr.$AWS_REGION.amazonaws.com/$AWS_ECR_REPOSITORY:$DOCKER_IMAGE_VERSION"
```

Comment:
yes? https://github.com/ipfs-elastic-provider/shared-workflows/issues/1

Reply:
Correct! :)
Glad you spotted that, forgot to mention when we were talking

Reply:
And I agree about moving this to template